products/sle/main.pm

```diff
@@ -380,7 +380,10 @@ sub load_x11regression_documentation {
         loadtest "x11regressions/libreoffice/libreoffice_mainmenu_components";
         loadtest "x11regressions/libreoffice/libreoffice_recent_documents";
         loadtest "x11regressions/libreoffice/libreoffice_default_theme";
-        loadtest "x11regressions/libreoffice/libreoffice_pyuno_bridge";
+        loadtest "x11regressions/evolution/evolution_prepare_servers";
+        if (get_var("VERSION") =~ /12/) {
```

Comment:
evolution_prepare_servers will fail in case of  >12 with unsupported system information.
I just want skip libreoffice_pyuno_bridge in this case, in same way as evolution tests are skipped to avoid failure following tests (and avoid confusion of reviewing person).